wiki/ALVR-client-and-streamer-on-separate-networks.md

```diff
@@ -106,15 +116,15 @@ ALVR version Experimental v7 or newer is recommended for this configuration.
 This configuration is **NOT** supported in ALVR v12. The latest release to still support this is v11.
 
 To run ALVR client and ALVR streamer on separate networks (broadcast domains) the following things must be done: 
-1. UDP ports 9943 and 9944 of ALVR streamer must be accessible from Oculus Quest device (i.e. firewall openings must be made to allow Oculus Quest to connect to ALVR streamer UDP ports 9943 and 9944). 
-1. Oculus Quest must be connected to computer and command-line `adb shell am startservice -n "com.polygraphene.alvr/.ChangeSettings" --es "targetServers" "10.10.10.10"` must be run in Command Prompt to specify IP address of ALVR streamer (`10.10.10.10` must be substituted with IP address of ALVR streamer; the long line is a single command-line).
-1. Next time when ALVR client will be started it should try to connect to the specified ALVR streamer. ALVR streamer should display the client in _Server_ tab (the same way local-network clients are displayed).
 
+1. UDP ports 9943 and 9944 of ALVR streamer must be accessible from Oculus Quest device (i.e. firewall openings must be made to allow Oculus Quest to connect to ALVR streamer UDP ports 9943 and 9944). 
+2. Oculus Quest must be connected to computer and command-line `adb shell am startservice -n "com.polygraphene.alvr/.ChangeSettings" --es "targetServers" "10.10.10.10"` must be run in Command Prompt to specify IP address of ALVR streamer (`10.10.10.10` must be substituted with IP address of ALVR streamer; the long line is a single command-line).
```

Comment:
Is this correct @zarik5 ?
Should it not be com.alvr-org.alvr or whatever namespace we use now or should this just be removed

Reply:
Oh, the client didn't support this for a long time lol. I think this was removed inv12

Reply:
I think we could remove the whole section `# ALVR v11 and Below`

Reply:
I think we could remove the whole section # ALVR v11 and Below

Reply:
Removed whole section